### ISSUE TITLE
If you change the names of the orgs in custom spec.yaml minfab up command fails

### ISSUE THREAD

**Issue by vipinsun** [hyperledger-labs/minifabric#279]
Verify the organization option ******************************
  **Default option org is org0.example.com which does not exist in your spec, use                                                                              -o to specify one**

# STATS *******************************************************
minifab: ok=19  failed=1

Specifying -o with an existing org name (in the custom spec.yaml works)

The failure is probably due to the hard-coded default org name only to be overridden by the -o flag

This failure has knock-on effects as the minifab cleanup does not work either and you have to manually remove the files

**Comment by vipinsun** [comment]
I see that if you have your custom envsettings to line up with spec.yaml, this does not happen.
envsettings can also be customized, it seems

**Comment by litong01** [comment]
That is right. Guess what the user really wants to do is hard. It is really better leaving this to user to specify the org since user already customized the spec and the error msg clearly indicates what is wrong. I think there is no need to do extra on this.

**Comment by vipinsun** [comment]
Other effects are not being able to do ccup as the code fails with null channel name (even though channel name is in the envsettings)

**Comment by vipinsun** [comment]
There are legitimate reasons for taking overrides from environment. That is why spec.yaml has been created. This philosophic differences are in every configuration. I agree that if -o fixes the problem this should not be top of mind.
